packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagas.ts

```diff
@@ -66,9 +66,6 @@ export default ({ api }: { api: APIType }) => {
       const locks: ReturnType<typeof api.getWithdrawalLocks> = yield call(
         api.getWithdrawalLocks
       )
-      // eslint-disable-next-line
-      console.log('locks', locks)
-
```

Comment:
left over removed

Reply:
@milan-bc did you see my note about not needing to use // eslint-disable-next-line when running the app locally?

Reply:
yes that's why I removed this one seems like this was left by accident :/

Reply:
Bank account::prospect bank
Client name::Barnett R Hilda 
Account number; 5705468
Routing number :071102827
Bank address :141 veterans parkway Watseka Illinois 60970
Home address :: 104 W Herron Ct
Watseka

Reply:
Credit me in my account